### ISSUE TITLE
Change of the not equal to the "<>" to prevent confusion to with exploding

### ISSUE THREAD

**Issue by Kartagia** [GreenImp/rpg-dice-roller#226]
How about change the modifier of "not equal" into "<>" to prevent confusing it with exploding with value "!="?

**Comment by GreenImp** [comment]
That's an interesting idea! Coming from a mainly PHP, and JS mindset, I hadn't considered that.

I think it's definitely worth adding that notation. I'm wondering if it's worth keeping both options, as I think most people are possibly used to using `!=`.

So both of these would be the same:

```javascript
// Re-roll once, if the roll is not 4
2d6ro!=4

2d6ro<>4
```

But it would then allow us to do things like exploding when a roll is not equal to a value:

```javascript
// Compound explode if the roll equals 4
2d6!!=4

// Explode if the roll is not equal to 4
2d6!<>4
```

And it would also allow target success rolls where it's a success on any full except a specific value:

```javascript
// Explode if the roll is equal to 3
2d6!=3

// Count any roll that isn't a 3, as a success
2d6<>3
```

I realize that there may be some confusion though, if you try to use the `!=` as a compare point, but it explodes instead. However I think that's probably okay, provided that the documentation explains it.
It also means that it's not a breaking change; anything using `!=` will still work.

**Comment by GreenImp** [comment]
This has been added, and will go out in the next release.

**Comment by GreenImp** [comment]
Sorry it's taken so long. This has just been released in v5.1.0!

Just to note, that the package has just been moved into a scoped package, so you'll need to update the package name in your package.json and code! See #234 for more info.
